### PR TITLE
add saving to db

### PR DESCRIPTION
**Pull request by soaptherope**

### CODE REVIEW

**Comment by soaptherope** [comment]
if i add tests for checking db, will they be able to pass in github actions?

**Comment by soaptherope** [comment]
doesnt go through actions because db is closed

**Comment by soaptherope** [comment]
but all tests pass on local machine

**Comment by soaptherope** [comment]
okay, all tests pass now =)

check when you can please

File: .idea/dbnavigator.xml
Comment (Alexmansar) [inline .idea/dbnavigator.xml:1]: remote it please))

**Review by Alexmansar — APPROVED** [review]
@soaptherope hI)) As i can see - good job)) remove code that we don't need here

**Pull request merged**